Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -222,6 +222,23 @@ such that `z = f x * (f y)⁻¹`. -/
 noncomputable def lift (hg : Injective g) : K →+* L :=
   IsLocalization.lift fun y : nonZeroDivisors A => isUnit_map_of_injective hg y
 
+theorem lift_unique
+    {L: Type*} [Field L]
+    {g: A →+* L} (hg: Function.Injective g)
+    {f: K →+* L}
+    (hf1: ∀ x, f (algebraMap A K x) = g x): IsFractionRing.lift hg = f :=
+  IsLocalization.lift_unique _ hf1
+
+/-- Another version of unique to give two lift maps should be equal -/
+theorem lift_unique'
+    {L: Type*} [Field L]
+    {g: A →+* L} (hg: Function.Injective g)
+    {f1 f2: K →+* L}
+    (hf1: ∀ x, f1 (algebraMap A K x) = g x)
+    (hf2: ∀ x, f2 (algebraMap A K x) = g x): f1 = f2 :=
+  Eq.trans (lift_unique hg hf1).symm (lift_unique hg hf2)
```

Comment:
Is this really useful? I mean `rw [lift_unique' hg hf1 hf2]` can be just `rw [← lift_unique hg hf1, lift_unique hg hf2]`.

Reply:
> Is this really useful? I mean `rw [lift_unique' hg hf1 hf2]` can be just `rw [← lift_unique hg hf1, lift_unique hg hf2]`.

For example. if the goal is `f1.comp f2 = g1.comp g2`, you can direcly `apply lift_unique'` and then prove the conditions, but `rw` may not find the correct function (unless we prove the conditions first and pass to it).

Reply:
OK.